common-files/utils/ethereum/gas.mjs

```diff
@@ -0,0 +1,32 @@
+import logger from '../logger.mjs';
+
+export const TX_GAS_DEFAULT = 4000000;
```

Comment:
I think some of these constants already exist [here](https://github.com/EYBlockchain/nightfall_3/blob/master/cli/lib/constants.mjs).

Reply:
@israelboudoux, @imagobea did the right thing. We should not be importing constants from another top-level package. Alternatively we can take from from config. Personally I would keep it as is. Two repetitions doesn't hurt my DRY-mania :)